CASSANDRA-17696 WEBSITE - Updating Ecosystem information for Anant

adp8ke opened the pull request:
Updating Ecoystem webpage for Anant

adp8ke committed:
WEBSITE - Updating Ecosystem information for Anant

nonstopdtop committed:
BLOG - New SSTable Identifiers in 4.1

patch by Jacek Lewandowski, Chris Thornett, Diogenese Topper; reviewed by Erick Ramirez for CASSANDRA-17692

Co-authored by: Jacek Lewandowski <lewandowski.jacek@gmail.com>
Co-authored by: Chris Thornett <chris@constantia.io>
Co-authored by: Diogenese Topper <diogenese@constantia.io>

michaelsembwever commented on site-content/source/modules/ROOT/pages/ecosystem.adoc:
```diff
@@ -69,7 +69,9 @@ https://ignite.apache.org/[Apache Ignite,window=blank]: Use Ignite as a traditio
 https://github.com/Azure-Samples/cassandra-proxy[Azure-Samples/Cassandra Proxy,window=blank]: This proxy handles client connections and forwards them to two Cassandra clusters simultaneously.
 
-http://cassandra.link/[Cassandra.link,window=blank]: Curated site with tools, along with cassandra.tools.
+http://cassandra.link/[Cassandra.Link,window=blank]: Curated site with Cassandra articles, videos, and news feed.
```
with the addition of `cassandra.tools` it means that `cassandra.link` doesn't really belong in the "Cassandra Tools" section anymore. In fact I'm not sure where we would put `cassandra.link` on this page anymore?

michaelsembwever commented on site-content/source/modules/ROOT/pages/ecosystem.adoc:
```diff
@@ -346,6 +350,8 @@ https://opencredo.com/about-us/[Open Credo,window=blank]
 [openblock,accordion-content ml-medium hidden]
 --------
 
+https://youtube.com/playlist?list=PLmZzyjM-vqX6f0WQYhHgIv5K-esMRcbyr[Apache Cassandra Lunch,window=blank]
```
This is more content, than formal education (workshops, courses, etc). Like `cassandra.link` I am not sure where we should list this, open to ideas but the current list of categories doesn't really cater for either…?

adp8ke replied:
@michaelsembwever Would you be open to creating a new category called "Cassandra Community Resources" or "Cassandra Community"? And we can put cassandra.link and Cassandra lunch into that category?

michaelsembwever replied:
@ErickRamirezAU what are your thoughts? what about "Community Blogs and Content" ?

ErickRamirezAU replied:
> @ErickRamirezAU what are your thoughts? what about "Community Blogs and Content" ?

I think the [Learn > Resources](https://cassandra.apache.org/_/resources.html) section of the website is a better fit for this. 🙂

michaelsembwever replied:
@ErickRamirezAU , are you thinking that Anant submits two tiles here (one for `cassandra.link` and one for the youtube playlist)? or that they submit each content and video piece they have as separate tiles on that page?

ErickRamirezAU replied:
+1 on doing 2 separate tiles. 👍

michaelsembwever replied:
@adp8ke are you good with that?

adp8ke replied:
@michaelsembwever Acknowledged, I am good with that. I will make the updates. Thank you!

michaelsembwever reviewed (changes requested):
Two comments open regarding where the appropriate place for cassandra.link and the youtube playlist should be.

Other than that it looks good to me.

ErickRamirezAU commented:
Noting here that I've logged [CASSANDRA-17696](https://issues.apache.org/jira/browse/CASSANDRA-17696) for this PR. 🍻

adp8ke committed:
Moving Cassandra.Link and Cassandra Lunch to Learn > Resources

adp8ke commented:
@michaelsembwever I have moved Cassandra.Link and Cassandra Lunch to Learn > Resources

nonstopdtop committed:
BLOG - Authentication Plugin Support for CQLSH in 4.1

patch by Brian Houser, Chris Thornett, Diogenese Topper; reviewed by Erick Ramirez for CASSANDRA-17703

Co-authored by: Brian Houser <bhouseapache@gmail.com>
Co-authored by: Chris Thornett <chris@constantia.io>
Co-authored by: Diogenese Topper <diogenese@constantia.io>

michaelsembwever commented on site-content/source/modules/ROOT/pages/resources.adoc:
```diff
@@ -798,4 +798,42 @@ https://www.youtube.com/watch?v=td5QRYOeGag&list=PLU2OcwpQkYCy_awEe5xwlxGTk5UieA
+[openblock,title mt-medium]
+------
+[discrete]
+=== Cassandra.Link
```
something more descriptive please (the reader needs a reason to click on it)

adp8ke replied:
Acknowledged, I will make it more descriptive

michaelsembwever commented on site-content/source/modules/ROOT/pages/resources.adoc:
```diff
@@ -798,4 +798,42 @@ https://www.youtube.com/watch?v=td5QRYOeGag&list=PLU2OcwpQkYCy_awEe5xwlxGTk5UieA
+[openblock,title mt-medium]
+------
+[discrete]
+=== Apache Cassandra Lunch Playlist
```
```suggestion
=== Apache Cassandra Lunch Youtube Playlist By Anant
```

adp8ke committed:
Update site-content/source/modules/ROOT/pages/resources.adoc

Co-authored-by: mck <mck@apache.org>

adp8ke committed:
updating cassandra.link resource card

adp8ke commented:
@michaelsembwever following up on this in case it slipped through the cracks

adp8ke commented:
@michaelsembwever thank you for approving. Who should I contact regarding merging this PR?

nonstopdtop committed:
BLOG - Talks & Sponsors Announced for Cassandra World Party 2022

patch by Chris Thornett, Diogenese Topper; reviewed by Erick Ramirez for CASSANDRA-17720

Co-authored by: Chris Thornett <chris@constantia.io>
Co-authored by: Diogenese Topper <diogenese@constantia.io>

adp8ke commented:
Hello @ErickRamirezAU, I tried to log into jira to comment on the ticket, but the reset password link is not working, perhaps due to the current ongoing jira issues. Can this PR be merged, or is anything else required?

adp8ke committed:
Merge branch 'apache:trunk' into trunk

michaelsembwever commented:
@adp8ke the merge commit needs to be removed from the PR

(We don't merge in two directions. Only merge (fast-forward) PRs into trunk. To update your PR, rebase your work on top of the origin's trunk.)

adp8ke commented:
@michaelsembwever Acknowledged, I will update this PR

adp8ke committed:
Merge branch 'trunk' of https://github.com/adp8ke/cassandra-website into trunk

rebasing

adp8ke commented:
@michaelsembwever going to close this PR and create another. Apologies.
